.github/workflows/gradle.yml

```diff
@@ -39,5 +39,5 @@ jobs:
         uses: SkriptLang/skript-test-action@v1.0
         with:
           test_script_directory: src/test/scripts
-          skript_repo_ref: 2.8.2
+          skript_repo_ref: dev/patch
```

Comment:
Is this something we should be committing?

Reply:
yup. the test action has to compile skript and 2.8.2 is broken by the latest paper changes, so you currently have to use dev/patch in order for skript to compile against the latest paper. i'm thinking of ways to address this on the skript-test-action side, but havent implemented yet